.github/workflows/continuous-integration-workflow.yml

```diff
@@ -50,11 +65,13 @@ jobs:
       - name: maybe_use_external_gtest
         if: ${{ matrix.distro == 'ubuntu:latest' }}
         run: sudo apt-get update && sudo apt-get install -y libgtest-dev
-      - name: CMake
+      - name: Configure Kokkos
         run: |
           cmake -B builddir \
             -DCMAKE_INSTALL_PREFIX=/usr \
+            -Ddesul_atomics_ROOT=/usr/desul-install/lib/cmake/desul/ \
```

Comment:
```suggestion
            -Ddesul_atomics_ROOT=/usr/desul-install/ \
```

Reply:
That didn't work. I guess `CMake` expected the path to look like "/usr/desul-install/lib/cmake/desul_atomics".

Reply:
Yes we need to fix that on the diesel side...

Reply:
I would prefer addressing that in a follow-up [pull request, possibly including a pull request to desul.

Reply:
Yes sorry I did not mean to imply that is a blocker for this PR

Reply:
See https://github.com/desul/desul/pull/65.